apps/web/src/design-system/segmented-control/SegmentedControl.tsx

```diff
@@ -15,6 +16,9 @@ interface ISegmentedControlProps {
   value?: string;
   onChange?(value: string): void;
   loading?: boolean;
+  fullWidth?: boolean;
+  sx?: Sx | (Sx | undefined)[];
```

Comment:
@davidsoderberg what dodes `sx` means?

Reply:
`sx` is Mantines way of doing style prop